Server/App/policy/deck.py

```diff
@@ -11,8 +11,7 @@ def can_view(deck_id, user_id, share_code=None, connector=None):
             return None
 
         if (
-            not metadata['public'] and
-            (user_id is None or metadata['owner'] != user_id) and
+            metadata['accessible'] is False and
             (share_code is None or metadata['share_code'] != share_code)
```

Comment:
probably no longer need this line if accessible is supposed to encapsulate all the conditions?